eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/BroadCastSubClient.java

```diff
@@ -44,10 +42,12 @@ public static void main(String[] args) throws Exception {
                 if (msg.getHeader().getCommand() == Command.BROADCAST_MESSAGE_TO_CLIENT) {
                     if (msg.getBody() instanceof EventMeshMessage) {
                         String body = ((EventMeshMessage) msg.getBody()).getBody();
-                        LogUtils.info(log, "receive message -------------------------------{}", body);
+                        if (log.isInfoEnabled()) {
+                            log.info("receive message : {}", body);
+                        }
```

Comment:
By changing " -------------------------------" to ": ", there should not be any additional space between the word "message" and the colon ":".

Besides, why `LogUtils` was substituted with logger?